specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1566,11 +1566,17 @@
             ]
           }
         },
+        "messageType": {
+          "description": "The received WhatsApp message type",
+          "type": "string"
+        },
         "media": {
           "description": "The received message media content",
-          "$ref": "#/definitions/AcsMessageMediaContent",
-          "type": "object",
-          "x-ms-client-name": "mediaContent"
```

Comment:
We are removing an x-ms-client-name here?

Reply:
It shouldnt be. Changed it back.